templates/workspaces/base_workspace/terraform/network/network.tf

```diff
@@ -0,0 +1,34 @@
+resource "azurerm_virtual_network" "ws" {
+  name                = "vnet-${var.resource_name_prefix}-${var.environment}-${var.tre_id}"
+  location            = var.location
+  resource_group_name = var.resource_group_name
+  address_space       = [var.address_space]
+}
+
+
+resource "azurerm_subnet" "services" {
+  name                                           = "ServicesSubnet"
+  virtual_network_name                           = azurerm_virtual_network.ws.name
+  resource_group_name                            = var.resource_group_name
+  address_prefixes                               = [local.services_subnet_address_prefix]
+  enforce_private_link_endpoint_network_policies = true
```

Comment:
Can we add a comment here maybe to highlight that this settings means private endpoints do not adhere to NSG rules applied to them? I believe the TF provider has the incorrect naming which causes confusion.

Reply:
added